src/test/java/org/assertj/core/internal/files/File_assertHasSize_Test.java

```diff
@@ -57,8 +57,9 @@ void should_throw_error_if_actual_file_does_not_have_the_expected_size() {
     // GIVEN
     AssertionInfo info = someInfo();
     // WHEN
-    expectAssertionError(() -> files.assertHasSizeInBytes(info, actual, 36L));
+    AssertionError error = expectAssertionError(() -> files.assertHasSizeInBytes(info, actual, 36L));
     // THEN
+    assertThat(error).isInstanceOf(AssertionError.class);
```

Comment:
`expectAssertionError` already makes sure the error is an `AssertionError`, please revert all the changes in this file.
The only thing to do is remove @DisplayName("Files.assertHasSizeInBytes:") as it is computed automatically